api/XDS_PROTOCOL.md

```diff
@@ -281,6 +281,11 @@ static_resources:
         port_value: <ADS management server port>
     lb_policy: ROUND_ROBIN
     http2_protocol_options: {}
+    upstream_connection_options:
```

Comment:
Thanks for this, can you potentially also add this in the main docs around here? https://www.envoyproxy.io/docs/envoy/latest/configuration/overview/v2_overview#aggregated-discovery-service

@htuch a bunch of these issue have gotten my thinking that I wonder if we should have some type of app level keep alive message as part of xDS? Should we open an issue to think about that?

Reply:
Thanks, added! 👍 

I did consider (and started mocking a prototype) of a way to send keep alive messages. I wonder if it makes sense to piggyback on gRPC health checking since we should have it supported in most gRPC servers by default without updating the proto or having something custom (unless there is more information to be communicated beyond just health). Probably for another issue though.

Reply:
I think keep-alive is already possible server side today, since a management server can just bump system version with the same content. I think it would be reasonable to add this at the wire-level, so please do open an issue. Ideally we don't conflate with gRPC health check, as this could also be used for REST.